gridData/core.py

```diff
@@ -452,12 +521,9 @@ def _load(
                     grid, edges, origin, delta))
 
     def load(self, filename, file_format=None):
-        """Load saved (pickled or dx) grid and edges from <filename>.pickle
+        """Load saved grid and edges from `filename`
```

Comment:
Just checking that you want italics here and not double ticks around the argument name.

Reply:
Same for all the other below.

Reply:
Single tics ("no role") seems to be the standard when referring to arguments.